config/checkstyle/checkstyle.xml

```diff
@@ -0,0 +1,378 @@
+<?xml version="1.0"?>
```

Comment:
this file is very long, I think that should exists a recommended config like an import

Reply:
I check in the documentation, and this file is needed